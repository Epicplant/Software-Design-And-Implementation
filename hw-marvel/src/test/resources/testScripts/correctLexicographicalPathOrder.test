# Tests that if there are 2 paths of the exact same length from node 1 to node 2 that the lexicographically
# shortest path is used

# Loads the graph
LoadGraph heroes lexicographicGraphTestFile.csv

# Finds shortest path between heroes
FindPath heroes A D