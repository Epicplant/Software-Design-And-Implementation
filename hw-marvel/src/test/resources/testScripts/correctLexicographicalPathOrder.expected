# Tests that if there are 2 paths of the exact same length from node 1 to node 2 that the lexicographically
# shortest path is used

# Loads the graph
loaded graph heroes

# Finds shortest path between heroes
path from A to D:
A to B via 1
B to D via 4